src/Sylius/Behat/Context/Api/Shop/CheckoutContext.php

```diff
@@ -669,7 +669,7 @@ public function myOrdersShippingMethodShouldBe(ShippingMethodInterface $shipping
         $shippingMethodName = $shippingMethod->getName();
 
         foreach ($shippingMethods as $method) {
-            if ($method['shippingMethod']['translations']['en_US']['name'] === $shippingMethodName) {
+            if ($method['translations']['en_US']['name'] === $shippingMethodName) {
```

Comment:
I think we should mention that in UPGRADE file that the response schema has changed.

Reply:
Added 👌🏻